src/app/pages/LandingPage.svelte

```diff
@@ -28,8 +28,13 @@
     continueReady = languageStatus && downloadURL !== "";
   }
 
-  function updateLanguageStatus(event: CustomEvent) {
+  // TODO: rename function; it does more than update language status
+  function setLanguage(event: CustomEvent) {
```

Comment:
Could this be something that can be taken care of in this PR?

Reply:
Yes, haha! I'll need to do another refactoring pass in a new PR.